webservice/app/Http/Controllers/CategoriesController.php

```diff
@@ -42,11 +42,9 @@ public function fullList()
      */
     public function store(CategoryRequest $request)
     {
-        Category::create($request->only('name'));
+        $category = Category::create($request->all());
```

Comment:
@vedovelli I've used in this case because the $fillable will filter the fields, but I can change back if you think is better :)